base/system/services/database.c

```diff
@@ -1449,15 +1555,15 @@ ScmControlService(HANDLE hControlPipe,
                 if (bResult == FALSE)
                 {
                     dwError = GetLastError();
-                    DPRINT1("GetOverlappedResult() failed (Error %lu)\n", dwError);
+                    DPRINT("(%S, %d) GetOverlappedResult() failed (Error %lu)\n", lpService->lpServiceName, dwControl, dwError);
 
                     goto Done;
                 }
             }
         }
         else
         {
-            DPRINT1("ReadFile() failed (Error %lu)\n", dwError);
+            DPRINT("(%S, %d) ReadFile() failed (Error %lu)\n", lpService->lpServiceName, dwControl, dwError);
```

Comment:
Just a question out of curiosity: Why the DPRINT1->DPRINT in this error-scenario? Did it spam?

Reply:
It did not spam. In general, after these changes, the code seems to work well and there is not any need for printing detailed failure messages, so demoting the DPRINT1's to DPRINT's seemed reasonable.